libs/core/src/commonProtocol/chainConfig.ts

```diff
@@ -18,4 +19,9 @@ export const factoryContracts: {
     communityStake: '0x377004f12eEE739204D44073F160798235160711',
     chainId: 11155111,
   },
+  [ValidChains.Base]: {
```

Comment:
Since all our usages only use the ValidChains.base now, should we remove the Sepolia entry? Or do we intend to use it for something in the future?

Reply:
I could be useful in the future for any test configs we might want to set up as a FF so figured we would keep the record in there